### ISSUE TITLE
Confirm default styling requirements for paragraphs in localgov_campaigns

### ISSUE THREAD

**Issue by finnlewis** [localgovdrupal/localgov_campaigns#63]
Lots of great new paragraphs types are being added by @gosia-mlynarczyk from Microserve for Collin Stenning at Bracknell Forest. Initially these have minimal css within the [localgov_campaigns_paragraphs](https://github.com/localgovdrupal/localgov_campaigns/tree/master/modules/localgov_campaigns_paragraphs) module, just doing basic layout.

The default theme when installing the localgov profile is the [localgov_theme](https://github.com/localgovdrupal/localgov_theme).

Ideally all the paragraphs should look good and work well out of the box, so we need to confirm how these are meant to look and create relevent styling tickets for each paragraph.

@gosia-mlynarczyk do you have designs for how each paragraph will end up looking or are there examples of each in play on the legacy Bracknell site?

@cjstevens78 do you have an idea for what level of design or wireframing we'd need to address each of these in the [localgov_theme](https://github.com/localgovdrupal/localgov_theme)?

**Comment by gosia-mlynarczyk** [comment]
Hi Finn, thanks for creating this as a separate ticket and unblocking my pull requests :slightly_smiling_face: 

Like I said on another ticket, it's been decided that I should add those new paragraphs with no (or very minimal) styling, just enough to make the paragraphs functional, and styling would be done in the theme at a later date. The reason we're not adding any styling to the theme is a) I have no designs for those components, I only know how they're meant to look on the Bracknell site (replicate the styling used on the legacy site) and b) as we won't be using `localgov_theme` for the Bracknell site, it's been agreed that we won't be contributing to it.

I appreciate that 'minimal' or 'functional' styling is very subjective, but I have no designs showing how each paragraph should look out of the box, so I'm only adding styling that is necessary for the paragraph to work (like classes for showing / hiding panels in the accordion or tabs paragraphs). I guess we need to decide what's better - having almost no styling in the paragraphs module, which means that a theme used on the site can start with a clean slate, or adding some styling, making the paragraphs look better out of the box, but potentially forcing the theme to then undo some of that default styling.

**Comment by finnlewis** [comment]
Hi @gosia-mlynarczyk 

We're working on the Lambeth site and starting to tease apart the designs for the 'campaign pages' (referred to as 'engagement pages' on Lambeth. 

The first component we're looking at is a block with image, title, text, button with link and text. Here's an example of three in a row:

![image](https://user-images.githubusercontent.com/326588/96699775-90c05000-1386-11eb-9669-3d235eb74275.png)

Which campaign paragraph would you say is the closest fit for this? 

Many thanks,

Finn

**Comment by gosia-mlynarczyk** [comment]
Hi @finnlewis ,

I'd say Featured teaser (or Featured teasers which is a group of Featured teaser paragraphs with a heading)

**Comment by finnlewis** [comment]
Thanks @gosia-mlynarczyk - I think a "Featured teaser" does make sense.
